src/hotspot/cpu/riscv/stubGenerator_riscv.cpp

```diff
@@ -6005,6 +6000,11 @@ static const int64_t right_3_bits = right_n_bits(3);
     if (UseAdler32Intrinsics) {
       StubRoutines::_updateBytesAdler32 = generate_updateBytesAdler32();
     }
+#endif // COMPILER2
+
+    generate_compare_long_strings();
+
+    generate_string_indexof_stubs();
```

Comment:
I think we can put this two under macro `COMPILER2` too. Then we can further remove check for macro `COMPILER2_OR_JVMCI` in this function. I don't think these stubs are ever used by JVMCI on this platform for now.

Reply:
> I think we can put this two under macro `COMPILER2` too. Then we can further remove check for macro `COMPILER2_OR_JVMCI` in this function. I don't think these stubs are ever used by JVMCI which is only partially implemented on this platform for now.

Thanks for your review. Fixed